package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clubhouse-go",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
This will not change the version for Go and would need to be tagged

Reply:
Yes, I'll tag it in GH after it has been merged like with v0.0.1.